cli/samples/datapipelines/readme.md

```diff
@@ -1,6 +1,6 @@
 # Creating a Data Pipeline
 
-Read about data pipeline details and requirements in the [Xi IoT Admin Guide](https://portal.nutanix.com/#/page/docs/details?targetId=Xi-IoT-Infra-Admin-Guide:Xi-IoT-Infra-Admin-Guide).
+Read about data pipeline details and requirements in the [Karbon Platform Services for IoT Admin Guide](https://portal.nutanix.com/#/page/docs/details?targetId=Xi-IoT-Infra-Admin-Guide:Xi-IoT-Infra-Admin-Guide).
```

Comment:
these links will change once KPS docs are active.

Reply:
Is this the correct link address to change to: https://portal.nutanix.com/page/documents/details?targetId=Karbon-Platform-Services-Admin-Guide:Karbon-Platform-Services-Admin-Guide?

Reply:
yeah seems correct

Reply:
updated